src/gesture/touchGestures.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
 			var g = this.makeGesture('gesturestart', e, {vector: this.gesture, scale: 1, rotation: 0});
 			dispatcher.dispatch(g);
 		}
+		e.preventDefault();
```

Comment:
Won't this prevent native page scrolling on touch devices (only ones that support gestures, as we're making the change here instead of in touch.js)?